Classes/DataProcessing/GalleryProcessor.php

```diff
@@ -205,10 +216,13 @@ protected function prepareGalleryData()
                                 (int)$this->processorConfiguration['autogenerate.']['lqip'] === 1) {
                             $fileObj['urlLqip'] = $this->getFileUtility()->processFile(
                                 $image,
-                                [
+                                array_merge(
```

Comment:
array_merge is overkill here, we could simply add to main array `fileExtension`

Reply:
Done.